deltacat/utils/resources.py

```diff
@@ -104,14 +107,28 @@ def _update_vcpus(self):
             self.used_vcpu_seconds = self.used_vcpu_seconds + float(
                 str(cluster_resources["CPU"] - available_resources["CPU"])
             )
-            self.total_vcpu_seconds = self.total_vcpu_seconds + float(
-                str(cluster_resources["CPU"])
-            )
-        else:
-            self.total_vcpu_seconds = self.total_vcpu_seconds + float(
-                str(cluster_resources["CPU"])
+
+        self.total_vcpu_seconds = self.total_vcpu_seconds + float(
```

Comment:
Should this be inside the above if ` if "CPU" not in cluster_resources`?

Reply:
I didn't get this comment.

Reply:
Is the indention here correct? Will this fail if cluster_resources don't have `"CPU"` field?

Reply:
Yes, it's correct. The above code will return if "CPU" is not in the list:

```
        if "CPU" not in cluster_resources:
            return
```

Reply:
I see, L 104-106 was hidden by the UI